src/kwlib.sh

```diff
@@ -577,12 +577,17 @@ function get_all_git_config()
   local config="$1"
   local scope="$2"
   local flag="$3"
+  local inside_git_repo='git rev-parse --is-inside-git-dir'
   local cmd='git config --get-all'
   local -A output
   local scp
 
   flag=${flag:-'SILENT'}
 
+  if [[ ! $($inside_git_repo 2> /dev/null) ]]; then
```

Comment:
We already have a function for handling this check; look at `is_inside_work_tree`. If you use it, you can delete `inside_git_repo` variable and change this conditional to something like:
`if ! is_inside_work_tree; then`